src/backend/utils/misc/guc.c

```diff
@@ -2784,16 +2763,27 @@ static struct config_int ConfigureNamesInt[] =
 	},
 
 	{
-		{"continuous_query_batch_size", PGC_SIGHUP, QUERY_TUNING,
+		{"continuous_query_num_batch", PGC_SIGHUP, QUERY_TUNING,
```

Comment:
This name seems slightly unintuitive. Maybe we should call these,

    continuous_query_batch_length
    continuous_query_batch_size